packages/react-storage/src/components/FileUploader/ui/FileList/FileList.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+
+import { ComponentClassName } from '@aws-amplify/ui';
+import { Alert, View } from '@aws-amplify/ui-react';
+
+import { FileStatus } from '../../types';
+import { FileControl } from './FileControl';
+import { FileListProps } from './types';
+
+export function FileList({
+  displayText,
+  files,
+  hasMaxFilesError,
+  isResumable,
+  onCancelUpload,
+  onDeleteUpload,
+  onResume,
+  onPause,
+  showThumbnails,
+  maxFileCount,
+}: FileListProps): JSX.Element | null {
+  if (files.length < 1) {
+    return null;
+  }
+
+  const { getMaxFilesErrorText } = displayText;
+  const headingMaxFiles = getMaxFilesErrorText(maxFileCount);
+
+  return (
+    <View className={ComponentClassName.StorageManagerFileList}>
```

Comment:
Need to update these classnames, right?

Reply:
Updated to use name referencing FileUploader